invenio/modules/records/views.py

```diff
@@ -65,6 +65,11 @@ def decorated(recid, *args, **kwargs):
         g.collection = collection = Collection.query.filter(
             Collection.name == guess_primary_collection_of_a_record(recid)).\
             one()
+        g.bibrec = Bibrec.query.get(recid)
+
+        record = get_record(recid)
```

Comment:
No, because it is line 66 that throws the exception.

Traceback:
```
File "/home/tind/.virtualenvs/invenio/local/lib/python2.7/site-packages/invenio/modules/records/views.py", line 66, in decorated
Collection.name == guess_primary_collection_of_a_record(recid)).
File "/home/tind/.virtualenvs/invenio/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2404, in one
raise orm_exc.NoResultFound("No row was found for one()")
```